src/main/webapp/electionInfo.html

```diff
@@ -68,6 +68,12 @@
                   <div id="address-error-wrapper">
                     <p style="display: none" id="address-error-text">Invalid address.</p>
                   </div>
+
+                  <div id="user-clarification-wrapper">
+                    <p><em>Your location helps us suggest addresses to you, and your address 
+                      helps us find your election information. <br> 
+                      We promise to never store or share your data. </em></p> 
```

Comment:
Not a lawyer, but I think generally we shouldn't be using this kind of language.